### PR TITLE
Params configuration

### PR DESCRIPTION
**Pull request by jrybinski**
Hi! 😄 
I added a possibility to configure params format. 🔥

### CODE REVIEW

File: packages/redux-api-params-middleware/src/middleware.js [hunk 1]
```diff
@@ -6,16 +6,19 @@ import { parse, stringify } from 'qs';
 import type { Dispatch } from 'redux';
 import type { Action } from './types';
 
-function parametriseEndpoint(endpoint, params) {
+function parametriseEndpoint(endpoint, params, options = { arrayFormat: 'indices' }) {
```
Comment (tkajtoch) [inline packages/redux-api-params-middleware/src/middleware.js]: Please use `parametrize` instead. We had a discussion about using the American spelling of words.

Comment (tkajtoch) [reply packages/redux-api-params-middleware/src/middleware.js]: Oh sorry, it's not your fault. I just noticed that it was there before. You don't have to change that in this PR

**Comment by jrybinski** [comment]
@stodan Why you suggest configuring in middleware? Do you see any pros of your approach?

**Comment by stodan** [comment]
@jrybinski because if you do need this option, then most likely you will need to set it for all actions, which would be easier to do once, when configuring the middleware.

**Comment by jrybinski** [comment]
@stodan When we use only one API or all APIs require the same format. When we need a different format for different APIs then better to have it in RSAA. But I agree on that, the configuration in middleware will be useful too because if even if we use only one API but it's different than default one then we need to add it to every RSAA.
I will implement it in the near future 😄

**Pull request merged**